GitTfs.Setup/GitTfs.Setup.csproj

```diff
@@ -117,20 +117,20 @@
   <UsingTask AssemblyFile="packages\WixSharp.1.0.22.3\build\SetEnvVar.dll" TaskName="SetEnvVar" />
   <Import Project="..\.paket\paket.targets" />
   <Choose>
-    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.0' Or $(TargetFrameworkVersion) == 'v4.5')">
+    <When Condition="$(TargetFrameworkIdentifier) == '.NETFramework' And ($(TargetFrameworkVersion) == 'v4.0' Or $(TargetFrameworkVersion) == 'v4.5' Or $(TargetFrameworkVersion) == 'v4.5.2')">
       <ItemGroup>
         <Reference Include="Microsoft.Deployment.WindowsInstaller">
           <HintPath>..\packages\WixSharp\lib\Microsoft.Deployment.WindowsInstaller.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
-        <Reference Include="WixSharp.UI">
-          <HintPath>..\packages\WixSharp\lib\WixSharp.UI.dll</HintPath>
+        <Reference Include="WixSharp">
+          <HintPath>..\packages\WixSharp\lib\WixSharp.dll</HintPath>
           <Private>True</Private>
           <Paket>True</Paket>
         </Reference>
-        <Reference Include="WixSharp">
-          <HintPath>..\packages\WixSharp\lib\WixSharp.dll</HintPath>
+        <Reference Include="WixSharp.UI">
+          <HintPath>..\packages\WixSharp\lib\WixSharp.UI.dll</HintPath>
```

Comment:
Yup. And VS kept doing this--so I just decided to check it in to stop the git noise.